src/routes/u/[playerId].svelte

```diff
@@ -139,6 +139,15 @@
       await fetcher(`/api/user/${player.id}/giveRole/${role}`, { withCredentials: true });
       refreshPlayerData({ forceRevalidate: true, softRefresh: true });
    }
+
+   let isOculusPlayer : boolean;
+   $: isOculusPlayer = !!$playerData?.profilePicture?.endsWith('oculus.png');
+
+   let playerId : string | null;
+   $: playerId = $playerData?.id ?? null;
```

Comment:
We're already checking if $playerData is not null before showing the PlayerLink component in the below html, I'm not sure why this is needed?

Reply:
Yeah, but the check is in the markup, not the script. And the markup is rendered only after the first pass of the script is run.

Reply:
It gets re-rendered when $playerData is updated

Reply:
Nope, in this case, it would never be rendered because it would end in an error.

![image](https://user-images.githubusercontent.com/1758707/143485548-3f746b2b-8982-4aa1-9ac3-eecf0ecb2720.png)